src/cli/cli_network_data.cpp

```diff
@@ -716,8 +726,16 @@ otError NetworkData::OutputBinary(bool aLocal)
  * 08040b02174703140040fd00deadbeefcafe0504dc00330007021140
  * Done
  * @endcode
- * @cparam netdata show [@ca{-x}]
+ * @code
+ * netdata show 0xdc00
+ * Prefixes:
+ * fd00:dead:beef:cafe::/64 paros med dc00
+ * Routes:
+ * Services:
+ * Done
+ * @cparam netdata show [@ca{-x} @ca{rloc16}]
```

Comment:
To confirm, this is either-or right?  You use -x or rloc16, but not both at the same time?

Reply:
Yes. Correct. We cannot filter based on RLOC16 when getting netdata as raw bytes (using `-x`).

Maybe can add `|` in between to make it more clear?

Reply:
Yes, please add a `|` to indicate only one can be used, like here: https://openthread.io/reference/cli/commands#parameters_34